packages/components/stories/recipes/composition.stories.tsx

```diff
@@ -34,28 +34,18 @@ import {
 	ToastQueue,
 	Tooltip,
 	TooltipTrigger,
-} from '../src';
+} from '../../src';
 
 const Container = (props: ComponentPropsWithoutRef<typeof Fragment>) => <>{props.children}</>;
 
 const meta: Meta<typeof Container> = {
-	title: 'Recipes/Composition',
 	component: Container,
-	parameters: {
-		chromatic: { pauseAnimationAtEnd: true },
-	},
-	decorators: [
-		(Story) => (
-			<div style={{ height: 'var(--lp-size-224)' }}>
-				<Story />
-			</div>
-		),
-	],
```

Comment:
Need these to accurately snapshot overlays in Chromatic

Reply:
How do I do that?

Reply:
oh I see, I need to add the param back

Reply:
Gotcha. Added back. Had ditched it because in storybook, absolutely positioned elements are seen and it looked better to just be the height of the initial in-the-document-flow content. Wish we could adjust the viewport size for chromatic snapshots, but not the story itself. bummer.